pkg/resources/task.go

```diff
@@ -587,35 +595,40 @@ func ReadTask(withExternalChangesMarking bool) schema.ReadContextFunc {
 				return warehouse.Name(), nil
 			}, nil),
 			func() error {
-				upperSchedule := strings.ToUpper(task.Schedule)
-				switch {
-				case strings.Contains(upperSchedule, "USING CRON"):
-					// We have to do it this was because we want to get rid of the prefix and leave the casing as is (mostly because timezones like America/Los_Angeles are case-sensitive).
-					// That why the prefix trimming has to be done by slicing rather than using strings.TrimPrefix.
-					cron := task.Schedule[len("USING CRON "):]
-					if err := d.Set("schedule", []any{map[string]any{
-						"using_cron": cron,
-					}}); err != nil {
-						return err
-					}
-				case strings.Contains(upperSchedule, "MINUTE"):
-					minuteParts := strings.Split(upperSchedule, " ")
-					minutes, err := strconv.Atoi(minuteParts[0])
+				managedInitialWarehouseSizeIdx := slices.IndexFunc(taskParameters, func(p *sdk.Parameter) bool {
```

Comment:
I still don't get why we need to handle this parameter in other way than the others (e.g. why we do not handle it like the parameters in warehouse resource)

Reply:
removed this one, it's handled in the common parameter function